packages/vscode-extension/src/handlers.ts

```diff
@@ -369,6 +369,27 @@ export function addFileSystemWatcher(workspacePath: string) {
   backupConfigWatcher.onDidCreate(async (event) => {
     await openBackupConfigMd(workspacePath, event.fsPath);
   });
+
+  const packageLockFileWatcher = vscode.workspace.createFileSystemWatcher("**/package-lock.json");
```

Comment:
I think we need to monitor `package-lock.json` only when it's a teamsfx project, does it make sense?